tests/ctest/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.14.7)
+project(manylinux_ctest)
+include(CTest)
+
+# SQLite3 test Derived from CMake unit test for FindSQLite3
+# https://gitlab.kitware.com/cmake/cmake/-/tree/master/Tests/FindSQLite3/Test
+find_package(SQLite3 REQUIRED)
```

```suggestion
find_package(SQLite3 3.34 REQUIRED)
```

Check for 3.34+ to ensure the manylinux built version is found. This won't need to be updated each time sqlite is updated.